### DISCUSSION TITLE
🚀 Tell us what you want to know more in MLOps 🚀

### DISCUSSION THREAD

**Discussion by lalawuu** [tensorchord/envd#1224] [Newsletter]
Here is a long-term thread for sourcing good project/feature/tech in the MLOps area.

We love hearing about your interesting projects! Whether it's a craft, an invention, or something else that's creative.

Just leave the keyword you want to know more about or feel interested in here. We will give you a big surprise! 🎁

**Comment by gaocegege** [comment]
Some MLOps startups:

- https://www.union.ai/
- https://valohai.com/
- https://www.modular.com/
- https://arize.com/
